sdk/opendp/whitenoise/synthesizers/mwem/mwem.py

```diff
@@ -150,7 +178,7 @@ def initialize_A(self, histogram, dimensions):
         A += value
         return A
 
-    def histogram_from_data_attributes(self, data):
+    def histogram_from_data_attributes(self, data, splits=[]):

```

Comment:
Do we need to expose the non 
.fit
.sample

functions? If not, can we make them _histogram_from_data_attributes(...). That way we can keep the complexity away from the users

Reply:
good idea